.goreleaser.yml

```diff
@@ -0,0 +1,93 @@
+project_name: cranlogs
+
+release:
+  # in case there is an indicator for this in the tag e.g. v1.0.0-rc1
+  # If set to true, will mark the release as not ready for production.
+  # Default is false.
+  prerelease: auto
+
+before:
+  hooks:
+    - go mod tidy
+    - ./scripts/completions.sh
+    - ./scripts/manpages.sh
+
+builds:
+  - 
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - windows
+      - darwin
+      - linux
+    goarch:
+    - amd64
+    - arm64
+    - arm
```

Comment:
this is for non64 bit arm platforms, like raspberry pi, and below goarm: 7 is signaling the arch target - https://go.dev/doc/install/source#environment